docs/home/feature-management/features-and-variables/variables-and-variations.md

```diff
@@ -21,15 +21,15 @@ This will lead the user to a table containing all of the Variables used by this
 
 Each Feature contains its own set of Variables which it manages. **By default, upon creation of a Feature, a Boolean Variable will be created which has the same name as the Feature's key for easier reference.** Depending on the Feature type, the default Variations will be pre-set. The most common of which will be the Variations of "Variation OFF" and "Variation ON", with the boolean Variable being set to false and true, respectively.
```

Comment:
should probably change to 

"Each Feature manages its own set of Variables." instead of "Each Feature contains its own set of Variables which it manages."